### PR TITLE
Colour messages to logger

### PR DESCRIPTION
**Pull request by dkazanc**
Slightly decorated the logger output messages with different colours, hope its helpful: 
image

### CODE REVIEW

File: httomo/utils.py [hunk 1]
```diff
@@ -61,16 +61,26 @@ def log_once(output: Any, level: int = logging.INFO) -> None:
...
             logger.warning(output)
         else:
-            logger.info(output)
+            # logger.info(output)
+            if "section" in output:
+                logger.opt(ansi=True).info("<cyan>{}</cyan>".format(output))
+            elif "pattern" in output:
+                logger.opt(ansi=True).info("<green>{}</green>".format(output))
+            elif "rotation" in output:
+                logger.opt(ansi=True).info("<yellow>--->{}</yellow>".format(output))
```
Comment (yousefmoazzam) [inline httomo/utils.py:77]: Small nitpick: putting the `--->` here makes the arrow not be indented within the section logging like the other stuff logged within a section.

Would it be ok to remove the arrow from here, leave the stuff in `utils.py` for adding colour, and add the arrow to the CoR log message in `rotation.py`, so then the CoR log message is indented correctly?

Comment (dkazanc) [reply httomo/utils.py:77]: Thanks Yousef, good point. cheers

Comment (yousefmoazzam) [reply httomo/utils.py]: Haha sorry, I should have been more specific :sweat_smile: 

With the latest change, it's going to look like:
```
Section 0 (pattern=projection) with the following methods:
    data_reducer (httomolib)
    find_center_vo (httomolibgpu)
    remove_outlier (httomolibgpu)
    normalize (httomolibgpu)
--->The center of rotation is 79.5
    Finished processing last block
```

What I was suggesting was
```
Section 0 (pattern=projection) with the following methods:
    data_reducer (httomolib)
    find_center_vo (httomolibgpu)
    remove_outlier (httomolibgpu)
    normalize (httomolibgpu)
    ---> The center of rotation is 79.5
    Finished processing last block
```

It's not a big deal either way, just wanted to clarify what I was suggesting :slightly_smiling_face:

Comment (dkazanc) [reply httomo/utils.py]: OK, sure :)

**Pull request merged**